lambda/tests/data/ggcanary_lambda_parameters.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "parameters": {
+      "dest_email_address": "ggtruesecrets@protonmail.com",
```

Comment:
Could you change this email and the following ?

Reply:
Done.